tests/suites/coslite/cl.sh

```diff
@@ -0,0 +1,57 @@
+# Ensure that COS Lite deploys successfully, the
+# relations are setup as expected and we can access the dashboards.
+run_deploy_coslite() {
+	echo
+
+	local model_name file overlay_path admin_passwd alertmanager_ip grafana_ip prometheus_ip
+	model_name="deploy-coslite"
+	file="${TEST_DIR}/${model_name}.log"
+	ensure "${model_name}" "${file}"
+
+	overlay_path="./tests/suites/coslite/overlay"
+	juju deploy cos-lite --trust --channel=edge --overlay "${overlay_path}/offers-overlay.yaml" --overlay "${overlay_path}/storage-small-overlay.yaml"
+
+	echo "Wait for all coslite bundle charms to be active"
+	wait_for 0 '[.applications[] | select(."application-status".current != "active")] | length' 1800
```

Comment:
I think you should probably use `idle_status` instead of active here:
When I ran through this, polling completed with a unit still executing:
```
  | Unit             Workload  Agent      Address       Ports  Message
    | alertmanager/0*  active    idle       10.1.130.212         
    | grafana/0*       active    executing  10.1.130.219         
    | loki/0*          active    idle       10.1.130.255         
    | prometheus/0*    active    idle       10.1.130.204         
    | traefik/0*       active    idle       10.1.130.236         
```

Reply:
Oh yeah nice catch.

Reply:
Oh yeah nice catch.

Reply:
I think idle => active but I'm not certain

Reply:
I addressed this by using `active_condition` and I also noticed grafana caused the destroy_model to time out because it errors when removing it and remains in that state. Fixed it by appending `--force` when removing. I think it is something with grafana that intermittently caused an error state in the other charms.